### ISSUE TITLE
Center title

### ISSUE THREAD

**Issue by y-neck** [y-neck/couchbreak.ch#8]
```
                    <p id="registriere">REGISTRIERE</p> <!-- TODO:Center @Bellielle-->
```

https://github.com/y-neck/couchbreak.ch/blob/ed2b5aec4d008d4fb5b73b8b5f4146e3f3d7963c/subpages/registrieren.html#L52

**Comment by y-neck** [comment]
Fixed by @Bellielle via commit 74180dc at L. 474

**Comment by y-neck** [comment]
Fixed by @Bellielle via commit https://github.com/y-neck/couchbreak.ch/commit/74180dc11afe2c6ec15ebae1a4d057c7a2864d6b at L. 474
